Gruntfile.js

```diff
@@ -135,6 +135,25 @@ module.exports = function(grunt) {
     jasmine_node: {
       integration: ['spec/integration/']
     },
+    integration: {
+      all: {
+        options: {
+          template: 'node_modules/freedom-for-chrome/spec/helper/',
+          spec: ['spec/chrome_integration/gtalk.spec.js'],
+          helper: [
+            // "include: true" is needed for dependencies that are used by
+            // the jasmine tests in the core environment.
+            {path: 'dist/social.google.json', include: false},
```

Comment:
I think we do want dist/*.js right? Why would they clobber each other? It's up to the manifest file to choose which one it wants. Thats why we also have 3 manifests, social.google.json, social.facebook.json, and social.xmpp.json